content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -188,12 +198,11 @@ First, make sure that the recovery path is set correctly on all of the
 nodes in the cluster. If that does not resolve the issues, [contact our
 support team](mailto:support@redislabs.com).
 
-To recover all of the databases, run the rladmin command recover all.
+To recover all of the databases, run the command `rladmin recover all`.
 
-You can also recover the databases one by one, by using the rladmin
-command recover db \<id \| name\>.
+To recover the databases one by one, run the command `rladmin recover db <id|name>`.
```

Comment:
@Paz-Yanover Who needs to review this addition?

Reply:
@ushachar - who you want to review and approve it from your side?